src/pages/Register.js

```diff
@@ -42,6 +43,7 @@ export default class Register extends Component {
       })
       authStore.update(state => ({ ...state, registrationStatus: 'registered' }))
       await refreshTerraProfile()
+      Ajax().Metrics.captureEvent(Events.userRegister)
```

Comment:
Do we want any info about the user here? Or are we good without?

Reply:
Most of what's on the form is private info that we don't want in Mixpanel, and the anonymous contact email is automatically synced via the profile. So I think just the event is enough here.